.github/workflows/testing.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
     - name: Setup Go
       uses: actions/setup-go@v3
       with:
-        go-version: '>=1.18.0'
+        go-version: '>=1.20.0'
```

Comment:
If you want to upgrade the go version in CI it is done [here](https://github.com/openshift/release/blame/master/ci-operator/config/openshift/addon-operator/openshift-addon-operator-main.yaml#L6).

Yash added these workflows, but they are disabled in the OpenShift org so we should probably just remove them.

Reply:
Oh thank you for the hint, will open a PR in the release repo :)

Reply:
FYI: https://github.com/openshift/release/pull/37140

will continue here once it it though